### PR TITLE
Covid App

### PR DESCRIPTION
**Pull request by pountzas**
- A home page that shows general information about the Covid situation.
- A more information page that shows extra info about a single country the user selects.
- Unit tests for the reducer
- Integration tests for the main pages

### CODE REVIEW

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [x] [Required] Let's add a recorded video link to your project as requested in the project guidelines. 
- [ ] Let's make sure all the changes are present in the Pull Request you create. Currently, I cannot access all the files. 
- [x] Let's add a live demo link of your precious project in your readme. This will help anyone to access your project easily. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by pountzas** [comment]
Hello, I forgot to push the commit with a link on my readme. I don't understand what you mean that you can not access all the files...

File: README.md [hunk 1]
```diff
@@ -9,21 +9,13 @@
...
-<img width="763" alt="Leaderboard image" src="https://user-images.githubusercontent.com/58818795/132796694-8e77d562-6435-41f6-a354-6673e0ee3e00.png">
-
-<img width="763" alt="Leaderboard image" src="https://user-images.githubusercontent.com/58818795/132796737-9c16e670-90fb-4ca7-b7ca-98a409277a8d.png"> -->
-
-
 ## Video
 
-<!-- https://www.loom.com/share/86eeb7da0f744395a46ad417c81dad49 -->
-
+![link]https://www.loom.com/share/f89a97f5fa5f403dab880fd1b23ad4ee
```
Comment (Dipeshtwis) [inline README.md]: - [ ] As the previous reviewer requested, Please add a link to the deployed project in the README. As this is one of the main requirements of this capstone project.

Comment (Dipeshtwis) [inline README.md]: - Great that you have added the video presentation link.  :partying_face:

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @pountzas, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**

-------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (pountzas) [reply README.md]: I have messed up something with this file sorry I will link the live demo now.

File: README.md [hunk 2]
```diff
@@ -9,21 +9,17 @@
...
-
-
 ## Video
 
-<!-- https://www.loom.com/share/86eeb7da0f744395a46ad417c81dad49 -->
+![link]https://www.loom.com/share/f89a97f5fa5f403dab880fd1b23ad4ee
+
+## Live Preview
 
+![link]https://enigmatic-island-12726.herokuapp.com/
```
Comment (Dipeshtwis) [inline README.md]: - [ ] The Live link is giving me this error. Please fix it or deploy it on[ netlify](https://www.netlify.com/) if the error still persists.

Screenshot from 2021-09-24 23-51-43

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @pountzas, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**

-------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**